.travis.yml

```diff
@@ -55,7 +55,7 @@ jobs:
     - stage:
       <<: *tox
       addons:
-        firefox: latest-nightly
+        firefox: latest-dev
```

Comment:
Is there a bug on file for putting the tests back on Nightly? I'd like to see a comment here with that bug link.

Reply:
I opened https://github.com/mozilla/addons-frontend/issues/5781